libs/site-base/lang/en.json

```diff
@@ -235,6 +235,10 @@
     "defaultMessage": "Create Account",
     "description": "app.account.create.form.buttonCreate: Button for the create account form page"
   },
+  "bIBRtw": {
+    "defaultMessage": "Join our Beta",
+    "description": "Super-title on request access page"
```

Comment:
Is this a new type of title?

Reply:
Err... no - just short-hand for "Above the Title".  Looks a little silly now I read it